conda_build/skeletons/cpan.py

```diff
@@ -346,7 +346,8 @@ def get_core_modules_for_this_perl_version(version, cache_dir):
 # meta_cpan_url="http://api.metacpan.org",
 def skeletonize(packages, output_dir=".", version=None,
                 meta_cpan_url="https://fastapi.metacpan.org/v1",
-                recursive=False, force=False, config=None, write_core=False):
+                recursive=False, force=False, config=None, write_core=False,
+                installdirs="vendor"):
```

Comment:
This should be taken from `config` right?. Otherwise the CLI value is not used.

Reply:
Thanks for the review!
There is some indirection in how the CLI invokes `skeletonize`:
https://github.com/mbargull/conda-build/blob/bcb43db98f730ee92f7123b9caf897cc8230e171/conda_build/cli/main_skeleton.py#L60-L61
https://github.com/mbargull/conda-build/blob/bcb43db98f730ee92f7123b9caf897cc8230e171/conda_build/api.py#L268-L269
and `Config` accepts any non-listed key as attributes to itself:
https://github.com/mbargull/conda-build/blob/bcb43db98f730ee92f7123b9caf897cc8230e171/conda_build/config.py#L305-L307
In other words: If the CLI argument and `skeletonize` parameter have the same name, this works implicitly. Like with the `meta_cpan_url` for example.